src/connection/utils.ts

```diff
@@ -25,3 +27,11 @@ export enum ErrorCode {
   WC_MODAL_CLOSED = 'Error: User closed modal',
   CB_REJECTED_REQUEST = 'Error: User denied account authorization',
 }
+
+export function didUserReject(connection: Connection, error: any): boolean {
```

Comment:
Yes I think it can be, but since that will mean updating the didUserReject function there and potentially renaming `swapErrorToUserReadableMessage`, probably makes sense to do in a sep PR. Added a TODO w/ a [ticket](https://uniswaplabs.atlassian.net/browse/WEB-3279)